test/unit/c_api/grid/test_grid_c_api.cpp

```diff
@@ -43,7 +43,7 @@ TEST(GridTest, GridScaLAPACKOrdering) {
 #endif
 
 TEST(GridTest, GridDLAFOrdering) {
-  for (const auto [key, value] : ordering) {
+  for (const auto& [key, value] : ordering) {
```

Comment:
Just curious, was this a warning or compilation error? Did we miss testing something previously?

Reply:
A new warning, IIRC, with clang 18